ceph-salt-formula/salt/ceph-salt/cephbootstrap.sls

```diff
@@ -61,21 +62,6 @@ run cephadm bootstrap:
 
 {{ macros.end_step('Run "cephadm bootstrap"') }}
 
-{{ macros.begin_step('Configure cephadm MGR module') }}
```

Comment:
We may need to keep `configure ssh orchestrator` due to https://github.com/ceph/ceph-salt/pull/267.

Perhaps, run it on admin nodes to guarantee that we don't require a `bootstrap_minion` on upgrades. 
 
[1] https://github.com/ceph/ceph-salt/blob/master/ceph-salt-formula/salt/ceph-salt/ceph-admin.sls

Reply:
The configure cephadm mgr module has also been a part of the fresh deployment.  Won't this still be necessary?

Reply:
The current implementation assumes that:

1) The first time we execute `ceph-salt apply`, the `bootstrap_minion` is required. This applies to fresh install and upgrades from Nautilus.

2) After first `ceph-salt apply`execution, there's no need to keep `bootstrap_minion` because it's no longer needed (but we can keep it if we want). This role is needed only for the first execution.